flink-connector-aws-e2e-tests/flink-formats-avro-glue-schema-registry-e2e-tests/src/test/java/org/apache/flink/glue/schema/registry/test/GSRKinesisPubsubClient.java

```diff
@@ -1,113 +1,113 @@
-
-        return configs;
-    }
-
-    private GlueSchemaRegistrySerializationFacade createSerializationFacade(
-            final AwsCredentialsProvider credentialsProvider) {
-        return GlueSchemaRegistrySerializationFacade.builder()
-                .credentialProvider(credentialsProvider)
-                .glueSchemaRegistryConfiguration(
-                        new GlueSchemaRegistryConfiguration(getSerDeConfigs()))
-                .build();
-    }
-
-    private GlueSchemaRegistryDeserializationFacade createDeserializationFacade(
-            final AwsCredentialsProvider credentialsProvider) {
-        return GlueSchemaRegistryDeserializationFacade.builder()
-                .credentialProvider(credentialsProvider)
-                .configs(getSerDeConfigs())
-                .build();
-    }
-}
+///*
+// * Licensed to the Apache Software Foundation (ASF) under one or more
+// * contributor license agreements.  See the NOTICE file distributed with
+// * this work for additional information regarding copyright ownership.
+// * The ASF licenses this file to You under the Apache License, Version 2.0
+// * (the "License"); you may not use this file except in compliance with
+// * the License.  You may obtain a copy of the License at
+// *
+// *    http://www.apache.org/licenses/LICENSE-2.0
+// *
+// * Unless required by applicable law or agreed to in writing, software
+// * distributed under the License is distributed on an "AS IS" BASIS,
+// * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// * See the License for the specific language governing permissions and
+// * limitations under the License.
+// */
+//
+//package org.apache.flink.glue.schema.registry.test;
+//
+//import org.apache.flink.streaming.connectors.kinesis.testutils.KinesisPubsubClient;
+//
+//import com.amazonaws.services.schemaregistry.common.AWSDeserializerInput;
+//import com.amazonaws.services.schemaregistry.common.AWSSerializerInput;
+//import com.amazonaws.services.schemaregistry.common.configs.GlueSchemaRegistryConfiguration;
+//import com.amazonaws.services.schemaregistry.deserializers.GlueSchemaRegistryDeserializationFacade;
+//import com.amazonaws.services.schemaregistry.serializers.GlueSchemaRegistrySerializationFacade;
+//import com.amazonaws.services.schemaregistry.utils.AWSSchemaRegistryConstants;
+//import com.amazonaws.services.schemaregistry.utils.AvroRecordType;
+//import org.apache.avro.generic.GenericRecord;
+//import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+//import software.amazon.awssdk.services.glue.model.DataFormat;
+//
+//import java.nio.ByteBuffer;
+//import java.util.HashMap;
+//import java.util.List;
+//import java.util.Map;
+//import java.util.Properties;
+//import java.util.UUID;
+//
+///**
+// * Simple client to publish and retrieve messages, using the AWS Kinesis SDK, Flink Kinesis
+// * Connectors and Glue Schema Registry classes.
+// */
+//public class GSRKinesisPubsubClient {
```

Comment:
I think instead of disabling the e2e test in this PR, we could consider:

1. Migrate to flink-connector-aws-kinesis-streams as part of this PR so that we have these tests working?
2. OR Migrate before this PR as a separate story and pre-requisite to move to Flink 2.0?

Any thoughts?

Reply:
Resolved in PR https://github.com/apache/flink-connector-aws/pull/212